docs/src/resources/two-side-column.md

```diff
@@ -0,0 +1,940 @@
+
+```
+module OperatorExampleModule(
+  input         clock,
+  input         reset,
+  input  [31:0] x,
+  input  [31:0] y,
+  output [31:0] a,
+  output [31:0] b,
+  output [31:0] c,
+  output [31:0] d,
+  output [31:0] e,
+  output [31:0] f,
+  output [31:0] g,
+  output [31:0] h,
+  output [31:0] i,
+  output [31:0] j,
+  output [31:0] k,
+  output [31:0] l,
+  output [31:0] m,
+  output [31:0] n,
+  output [31:0] o,
+  output [31:0] p,
+  output [31:0] q,
+  output [31:0] r,
+  output [31:0] s,
+  output [31:0] t,
+  output [31:0] u,
+  output [31:0] v,
+  output [31:0] w,
+  output [31:0] xx,
+  output [31:0] yy,
+  output [31:0] z,
+  output [31:0] zz,
+  output [31:0] zzz,
+  output [31:0] zzzz,
+  output [31:0] zzzzz,
+  output        aa,
+  output        bb
+);
+  wire [32:0] _b_T = x - y; // @[main.scala 16:10]
+  wire [63:0] _d_T = x * y; // @[main.scala 18:10]
+  wire  _r_T_2 = x[0] & y[0]; // @[main.scala 32:13]
+  wire  _s_T_2 = x[0] | y[0]; // @[main.scala 33:13]
+  wire [63:0] _t_T = {x,y}; // @[Cat.scala 30:58]
+  wire [38:0] _GEN_0 = {{7'd0}, x}; // @[main.scala 37:10]
+  wire [38:0] _w_T_1 = _GEN_0 << y[2:0]; // @[main.scala 37:10]
+  wire [95:0] _zzzz_T_1 = {x,x,x}; // @[Cat.scala 30:58]
+  wire [31:0] _GEN_1 = x % y; // @[main.scala 17:10]
+  wire [31:0] _GEN_2 = x % y; // @[main.scala 20:10]
+  assign a = x + y; // @[main.scala 15:10]
+  assign b = x - y; // @[main.scala 16:10]
+  assign c = _GEN_1[31:0]; // @[main.scala 17:10]
+  assign d = _d_T[31:0]; // @[main.scala 18:5]
+  assign e = x / y; // @[main.scala 19:10]
+  assign f = _GEN_2[31:0]; // @[main.scala 20:10]
+  assign g = x + y; // @[main.scala 21:10]
+  assign h = x - y; // @[main.scala 22:10]
+  assign i = _b_T[31:0]; // @[main.scala 23:5]
+  assign j = {{31'd0}, x == y}; // @[main.scala 24:5]
+  assign k = 32'h0;
+  assign l = {{31'd0}, x != y}; // @[main.scala 26:5]
+  assign m = x & y; // @[main.scala 27:10]
+  assign n = x | y; // @[main.scala 28:10]
+  assign o = x ^ y; // @[main.scala 29:10]
+  assign p = ~x; // @[main.scala 30:9]
+  assign q = {{31'd0}, x == 32'h0}; // @[main.scala 31:5]
+  assign r = {{31'd0}, _r_T_2}; // @[main.scala 32:5]
+  assign s = {{31'd0}, _s_T_2}; // @[main.scala 33:5]
+  assign t = _t_T[31:0]; // @[main.scala 34:5]
+  assign u = c[0] ? x : y; // @[main.scala 35:11]
+  assign v = x >> y[2:0]; // @[main.scala 36:10]
+  assign w = _w_T_1[31:0]; // @[main.scala 37:5]
+  assign xx = {{31'd0}, x > y}; // @[main.scala 38:6]
+  assign yy = {{31'd0}, x >= y}; // @[main.scala 39:6]
+  assign z = {{31'd0}, x <= y}; // @[main.scala 40:5]
+  assign zz = {{31'd0}, x[1]}; // @[main.scala 41:6]
+  assign zzz = {{30'd0}, x[1:0]}; // @[main.scala 42:7]
+  assign zzzz = _zzzz_T_1[31:0]; // @[main.scala 43:7]
+  assign zzzzz = {{31'd0}, &x}; // @[main.scala 44:9]
+  assign aa = |x; // @[main.scala 45:11]
+  assign bb = ^x; // @[main.scala 46:11]
+endmodule
+```
+</td>
+<td>
+
+```scala mdoc:silent
+class OperatorExampleModule extends Module {
```

Comment:
Can you create more descriptive `val` names for this module, so you don't have to re-letter all the names whenever this example is changed?

Like instead of 

```
  val a, b, ... = IO(Output(UInt(32.W)))
  
  a := x + y
  b := x - y 
```

You can make names like
```
  val add_res, sub_res, ... = IO(Output(UInt(32.W)))
  
  add_res := x + y
  sub_res := x - y 
```

Or you can use `sum`, `diff`, etc., whichever you prefer.

Reply:
okay... i will make all of that corrections and push the adjustments.

Reply:
I joined the meeting back everyone left already. till Wednesday I suppose?

Reply:
Yes, let's meet again then!